guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,48 @@
 
 In {Project}, you can use the package dependency resolution feature to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
 
-You can select to resolve package dependencies for any Content View that you want, or you can change the default setting to enable or disable resolving package dependencies for all new Content Views.
+You can select to resolve package dependencies per Content View or have it enabled by default via Settings -> Content -> "content view Dependency Solving Default".
```

Comment:
```suggestion
You can select to resolve package dependencies per Content View or have it enabled by default via *Settings > Content > "content view Dependency Solving Default" *.
```

Reply:
Please also replace "via" with something like "in".